charts/stable/kopia/docs/SERVER.md

```diff
@@ -0,0 +1,184 @@
+# Running Kopia as a Repository Server
+
+Kopia can run in a standalone mode to back up files from the computer on which
+it runs, or as a repository server to act as a repository for multiple computers
+to use remotely and share configuration. To be able to be used as a repository
+server, Kopia must be running in a secure mode with TLS certificates.
+
+## Instruction Summary
+
+1. When creating the Kopia App, use the preferred settings below
+2. Open the Kopia Web UI at `https://<server_ip>:10238/repo` and create a Local
+Directory or NAS repository at `/repository`
+3. Create a client user from the TrueNAS terminal by doing `sudo k3s kubectl exec -it <kopia_pod_name> -n ix-kopia -- kopia server user add <user>@<host>`
+using your pod's name and preferred user and host for the Kopia client. You will
+be prompted to create a password for the user which will be stored in the Kopia
+repository
+4. Get the `SERVER CERT SHA256` fingerprint from the Kopia logs with `sudo k3 kubectl logs -l pod.name=main -n ix-kopia`
+5. Configure your Kopia client with the same URL used for the Kopia Web UI and
+certificate fingerprint
+6. Enter the username and password you created when prompted by the Kopia client
+
+### Preferred Settings
+
+> - Username: kopia
+> - Protocol: HTTPS (TLS)
+> - Generate TLS Certificates: Checked -> Edit the app and uncheck after first start
+> - App Config Storage: Host Path -> Choose a dataset
+> - Additional App Storage: Host Path -> Choose a dataset -> Mount path: `/repository`
+
+## Detailed Instructions
+
+### Configure the App
+
+To use TLS certificates with the Kopia App change the "Protocol" option under
+"Main Service Configuration" to "HTTPS (TLS)". If you have certificates you wish
+to use, leave the "Generate TLS Certificate" option unchecked, and ensure you
+configure the host bindings to provide your certificate and private key at the
+paths specified. It is also recommended that you set the "User" parameter to
+"kopia" because of [this issue](https://github.com/kopia/kopia/issues/880#issuecomment-798421751).
+
+If you would prefer Kopia to generate certificates for you, check "Generate TLS
+Certificates" and run the app once. After it is running edit the application
+configuration and uncheck "Generate TLS Certificates" to allow the server to
+reuse the one it created on the first run. If you have Kopia generate the
+certificates you should consider having the Kopia configuration saved to a host
```

Comment:
Is the section where the certificates not mounted in the chart? If they aren’t then we should mount them. We don’t advise the use of hostpath for anything config, there’s easy ways of accessing the PVC. (Sorry dunno the app enough on mobile to comment more)

Reply:
The default path for those is in what gets mounted by default into the config PVC. My concern is that if you delete the app, you may want to keep the certs to move them to another instance, or to rebuild it from scratch on the same host. It's easy enough to update the docs to say to put them in another custom host volume in that case rather than the PVC.